omego/db.py

```diff
@@ -14,6 +14,43 @@
 
 log = logging.getLogger("omego.db")
 
+# Regular expression identifying a SQL schema
+SQL_SCHEMA_REGEXP = re.compile('.*OMERO(\d+)(\.|A)?(\d*)([A-Z]*)__(\d+)$')
```

Comment:
"OMERO" is not special in the name. It's just an arbitrary identifier. It's valid to have a sequence like:

`OMERO5.2__0 --> IDR1__0 --> IDR__2__0 --> OMERO5.3__0`

and in fact, we may soon.